tsconfig.settings.json

```diff
@@ -0,0 +1,22 @@
+{
```

Comment:
Is it possible to also activate `noUncheckedIndexedAccess`?

Reply:
Yes, we should definitely enable that. #643